tests/test_bgp_af.rb

```diff
@@ -28,31 +28,254 @@ def setup
     super
     # Disable and enable feature bgp before each test to ensure we
     # are starting with a clean slate for each test.
-    config('no feature bgp', 'feature bgp')
+    if platform == :nexus
+      config('no feature bgp', 'feature bgp')
+    elsif platform == :ios_xr
+      config('no router bgp')
+      config('no route-policy drop_all')
+    end
   end
 
-  def get_bgp_af_cfg(asn, vrf, af)
-    afi, safi = af
-    string =
-      @device.cmd("show run bgp all | sec 'bgp #{asn}' |  sec 'vrf #{vrf}' | " \
-                  "sec 'address-family #{afi} #{safi}' | no-more")
-    string
+  # rubocop:disable Style/WordArray
+  # rubocop:disable Style/ClassVars
+  # rubocop:disable Metrics/LineLength
```

Comment:
@glennmatthews Good call regarding Metrics/LineLength.  I was confusing it with the Metrics/MethodLength cop